src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,8 @@
 package core.basesyntax;
 
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/216778050-22710bbb-524b-4c8c-99b7-15619b4cf63e.png)

```suggestion
    public String getRandomColor() {
```